<launch>
    <node name="qnx2ros" pkg="message_transformer_cpp_track_ball" type="qnx2ros" output="screen" >
        <param name="filter_size" value="1" />
        <param name="is_vel_world" value="true" />
    </node>
    <node name="ros2qnx" pkg="message_transformer_cpp_track_ball" type="ros2qnx" output="screen" >
        <param name="vel_x_factor" value="1.0" />
        <param name="local_port" value="43894"/>
        <param name="remote_port" value="43893"/>
        <param name="remote_ip" value="192.168.1.120"/>
    </node>
</launch>
